### DISCUSSION TITLE
When is the next release for v4l2loopback

### DISCUSSION THREAD

**Discussion by sagi-ottopia** [umlaeute/v4l2loopback#576] [General]
Latest official release 0.12.7 was on Aug 5, 2022. Many useful features were added to the code, I can download the code and compile it myself but because there's no new tag, I might get into a conflict with the build coming out for my distribution. 
Thank you, Sagi.

**Comment by umlaeute** [comment]
`0.13.0` was released on Mar 19, 2024.
